<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerDialog</class>
 <widget class="QWidget" name="EqualizerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>314</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QWidget" name="EqualizerWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="activeCheckBox">
          <property name="text">
           <string>Active</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="eqPresetsGroupBox">
          <property name="title">
           <string>Presets</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="eqPresets">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>List of available equalizer presets.</string>
             </property>
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="eqPresetSaveBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Save changes of currently selected preset</string>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="eqPresetDeleteBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Delete currently selected preset. Not available for built-in presets.</string>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="eqPresetResetBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Reset selected preset setting to default or to the last saved change.</string>
             </property>
             <property name="text">
              <string>Reset to Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="eqBandsGroupBox">
          <property name="title">
           <string>Bands</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="eqMaxEq">
                 <property name="text">
                  <string>+dB</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="eqPreampValue">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="eqBand0Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="eqBand1Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QLabel" name="eqBand2Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QLabel" name="eqBand3Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="6">
                <widget class="QLabel" name="eqBand4Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="7">
                <widget class="QLabel" name="eqBand5Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="8">
                <widget class="QLabel" name="eqBand6Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="9">
                <widget class="QLabel" name="eqBand7Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="10">
                <widget class="QLabel" name="eqBand8Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="11">
                <widget class="QLabel" name="eqBand9Value">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <spacer name="verticalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::MinimumExpanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="eqZero">
                   <property name="text">
                    <string>0dB</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::MinimumExpanding</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item row="1" column="1">
                <widget class="QSlider" name="eqPreampSlider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="sliderPosition">
                  <number>0</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QSlider" name="eqBand0Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QSlider" name="eqBand1Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QSlider" name="eqBand2Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="5">
                <widget class="QSlider" name="eqBand3Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="6">
                <widget class="QSlider" name="eqBand4Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="7">
                <widget class="QSlider" name="eqBand5Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="8">
                <widget class="QSlider" name="eqBand6Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="9">
                <widget class="QSlider" name="eqBand7Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="10">
                <widget class="QSlider" name="eqBand8Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="11">
                <widget class="QSlider" name="eqBand9Slider">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="tickPosition">
                  <enum>QSlider::TicksAbove</enum>
                 </property>
                 <property name="tickInterval">
                  <number>20</number>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="eqMinEq">
                 <property name="text">
                  <string>-dB</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLabel" name="eqPreampLabel">
                 <property name="text">
                  <string>Preamp</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="eqBand0Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QLabel" name="eqBand1Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QLabel" name="eqBand2Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="5">
                <widget class="QLabel" name="eqBand3Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="6">
                <widget class="QLabel" name="eqBand4Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="7">
                <widget class="QLabel" name="eqBand5Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="8">
                <widget class="QLabel" name="eqBand6Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="9">
                <widget class="QLabel" name="eqBand7Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="10">
                <widget class="QLabel" name="eqBand8Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="11">
                <widget class="QLabel" name="eqBand9Label">
                 <property name="text">
                  <string>Hz</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
